lib/src/components/NxDescriptionList/NxDescriptionList.tsx

```diff
@@ -17,24 +18,26 @@ import { Props, propTypes } from './types';
 
 export { Props, ButtonItemProps, LinkItemProps } from './types';
 
-const _NxDescriptionList = forwardRef<HTMLElement, Props>(function({ emptyMessage, ...otherProps }, externalRef) {
-  const emptyChildRef = useRef<HTMLLIElement>(null),
-      listRef = useRef<HTMLElement>(null),
-      ref = useMergedRef(externalRef, listRef),
-      isEmpty = useEmptyComponent(listRef, emptyChildRef);
-
-  const emptyList = (
-    <ul className="nx-list" ref={ref} { ...otherProps }>
-      <li ref={emptyChildRef} className="nx-list__item nx-list__item--empty">
-        <span className="nx-list__text">
-          {emptyMessage || 'This list is empty.'}
-        </span>
-      </li>
-    </ul>
-  );
-
-  return isEmpty ? emptyList : <dl ref={ref} className="nx-list nx-list--description-list" { ...otherProps } />;
-});
+const _NxDescriptionList = forwardRef<HTMLElement, Props>(
+    function({ emptyMessage, className, ...otherProps }, externalRef) {
+      const classNames = classnames(className, 'nx-list'),
+          emptyChildRef = useRef<HTMLLIElement>(null),
+          listRef = useRef<HTMLElement>(null),
+          ref = useMergedRef(externalRef, listRef),
+          isEmpty = useEmptyComponent(listRef, emptyChildRef);
+
+      const emptyList = (
+        <ul className={classNames} ref={ref} { ...otherProps }>
+          <li ref={emptyChildRef} className="nx-list__item nx-list__item--empty">
+            <span className="nx-list__text">
+              {emptyMessage || 'This list is empty.'}
+            </span>
+          </li>
+        </ul>
+      );
+
+      return isEmpty ? emptyList : <dl ref={ref} className="nx-list nx-list--description-list" { ...otherProps } />;
```

Comment:
the `className` prop needs to be applied here too (right now it's just targeting the empty lists!)

Reply:
> the `className` prop needs to be applied here too (right now it's just targeting the empty lists!)

Nice catch! fixed here: 4bd43eeea473b69d4efce966d25eab8756a8876a